### ISSUE TITLE
Remove `include` support for `FilePropertiesEntries`

### ISSUE THREAD

**Issue by mr-archano** [novoda/gradle-build-properties-plugin#35]
Following the discussion in #27:

> I am also planning of removing this mechanism because not too flexible (relative paths provided by include property are now evaluated using the properties file parent folder as base). I really want to put in place something more explicit, [...] but embracing the inheritance of build properties holistically

```
buildProperties {
   default {
        from project.file('default.properties')
    }
    production {
        from project.file('production.properties')
        include 'default'
    }
}
```

**Comment by xrigau** [comment]
👍 great idea

**Comment by eximius313** [comment]
+1

**Comment by mr-archano** [comment]
Addressed by #46
